### ISSUE TITLE
How to enable error squiggles for java in vscode?

### ISSUE THREAD

**Issue by nikunjk9** [redhat-developer/vscode-java#3783]
Does this issue occur when all extensions are disabled?: Yes/No

VS Code Version:
OS Version:
Steps to Reproduce:

Just writing random things in editor. It doesn't show any errors.
![image](https://github.com/user-attachments/assets/eb1b112c-6248-483f-8542-547042fbdc55)

All this while error squiggles is still enabled. I have tried disabling and then enabling, uninstalling extensions and reinstalling. Nothing worked.

**Comment by jdneo** [comment]
How is your project layout looks like, is it using build tool? What extensions are enabled?

**Comment by nikunjk9** [comment]
> How is your project layout looks like, is it using build tool? What extensions are enabled?

No it is not using build tool, 

Following are the Extensions that are used:

![image](https://github.com/user-attachments/assets/a792e59c-ea88-4eb3-ae0f-6a47af162078)
![image](https://github.com/user-attachments/assets/68278ce4-e7c8-48b6-9d4e-1bde8e27117f)


Till yesterday I was able to see error squiggles while writing the code then suddenly It stopped appearing.
was it because of some extension update or anything else??

**Comment by jdneo** [comment]
@nikunjk9 would you mind sharing your project?

**Comment by nikunjk9** [comment]
My code contains nothing but DSA question practice. and this problem is not limited to a particular file or project it happens to each .java file. 

Earlier whenever I used to write the code in java I used to get these error squiggles (red line reflecting some error in code) which made the coding experience better.

![image](https://github.com/user-attachments/assets/93823673-c83a-45d3-8b07-d48d07467d17)


Now if I genuinely make an error in the code then it won't get this error squiggles 

![image](https://github.com/user-attachments/assets/f1708596-561c-40c4-8b50-d7807b3ddb11)


Apart from java file C/Cpp file are showing error squiggles. this issue is only with .java file
![image](https://github.com/user-attachments/assets/7f2c9ca0-62af-4d3a-93df-59c82fe06c9f)
